[2022-07-16 19:19:14,678] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:05:47.374925+00:00 [queued]>
[2022-07-16 19:19:14,688] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:05:47.374925+00:00 [queued]>
[2022-07-16 19:19:14,688] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-16 19:19:14,688] {taskinstance.py:1377} INFO - Starting attempt 4 of 4
[2022-07-16 19:19:14,688] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-16 19:19:14,701] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): setup_http_conn> on 2022-07-16 19:05:47.374925+00:00
[2022-07-16 19:19:14,707] {standard_task_runner.py:52} INFO - Started process 1802 to run task
[2022-07-16 19:19:14,710] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'process_files', 'setup_http_conn', 'manual__2022-07-16T19:05:47.374925+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/process_files.py', '--cfg-path', '/tmp/tmp5mc4pfhu', '--error-file', '/tmp/tmpna2tmqc_']
[2022-07-16 19:19:14,710] {standard_task_runner.py:80} INFO - Job 9: Subtask setup_http_conn
[2022-07-16 19:19:14,758] {task_command.py:371} INFO - Running <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:05:47.374925+00:00 [running]> on host 98d7be6daedb
[2022-07-16 19:19:14,818] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process_files
AIRFLOW_CTX_TASK_ID=setup_http_conn
AIRFLOW_CTX_EXECUTION_DATE=2022-07-16T19:05:47.374925+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-16T19:05:47.374925+00:00
[2022-07-16 19:19:14,819] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-16 19:19:14,819] {subprocess.py:74} INFO - Running command: ['bash', '-c', "\n*** connections add -a 'http_api'     --conn-type 'http'     --conn-host 'raw.githubusercontent.com/tadinve/EKG-Foundations/master/04A-postgress/     ; exit 99 "]
[2022-07-16 19:19:14,831] {subprocess.py:85} INFO - Output:
[2022-07-16 19:19:14,833] {subprocess.py:92} INFO - bash: -c: line 2: unexpected EOF while looking for matching `''
[2022-07-16 19:19:14,833] {subprocess.py:92} INFO - bash: -c: line 3: syntax error: unexpected end of file
[2022-07-16 19:19:14,833] {subprocess.py:96} INFO - Command exited with return code 2
[2022-07-16 19:19:14,847] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2022-07-16 19:19:14,850] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=process_files, task_id=setup_http_conn, execution_date=20220716T190547, start_date=20220716T191914, end_date=20220716T191914
[2022-07-16 19:19:14,859] {standard_task_runner.py:97} ERROR - Failed to execute job 9 for task setup_http_conn (Bash command failed. The command returned a non-zero exit code 2.; 1802)
[2022-07-16 19:19:14,881] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-07-16 19:19:14,918] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
